.travis.yml

```diff
@@ -0,0 +1,31 @@
+language:
+  - node_js
+
+node_js:
+  - 6
+
```

Comment:
can you add the travis cache directives here as well ex: https://github.com/smartcar/clutch-assert/blob/master/.travis.yml